README.md

```diff
@@ -300,39 +264,33 @@ it is possible to detect if an object is a presence.
 ### Handler traps
 
 A handler object can provide handler traps (`eventualGet`, `eventualApply`,
-`eventualSend`) and their associated `*Only` traps.
+`eventualSend`).
 
 ```ts
 ({
-  eventualGet        (target, prop):        Promise<result>,
-  eventualGetOnly    (target, prop):        void,
-  eventualApply      (target, args):        Promise<result>,
-  eventualApplyOnly  (target, args):        void,
-  eventualSend       (target, prop, args):  Promise<result>,
-  eventualSendOnly   (target, prop, args):  void,
+  eventualGet        (target, prop, opts):        Promise<result>,
+  eventualApply      (target, args, opts):        Promise<result>,
+  eventualSend       (target, prop, args, opts):  Promise<result>,
 })
 ```
 
-If the handler does not provide a `*Only` trap, its default implementation
-is the non-send-only trap with a return value of `undefined` (not a promise).
-
-If the handler omits a non-only trap, invoking the associated operation
-returns a promise rejection.  The only exception to that behaviour is if the
-handler does not provide the `eventualSend` optimization trap.  Then, its
-default implementation is
+If the handler omits a trap, invoking the associated operation returns a promise
+rejection.  The only exception to that behaviour is if the handler does not
+provide the `eventualSend` optimization trap.  Then, its default implementation
+is
 ```js
-Promise.eventualApply(Promise.eventualGet(p, prop), args)
+Promise.delegated.eventualApply(Promise.delegated.eventualGet(p, prop), args)
```

Comment:
We must decide what happens to the `opts` argument. I propose that it be passed to the `eventualApply` but not the `eventualGet`.